sqlalchemy_utils/types/enriched_datetime/pendulum_datetime.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+import six
+
+pendulum = None
+try:
+    import pendulum
+except ImportError:
+    pass
+
+
+class PendulumDatetime(object):
+    def _coerce(self, impl, value):
+        if value is not None:
+            if isinstance(value, pendulum.DateTime):
+                pass
+            elif isinstance(value, int):
```

Comment:
Sorry to jump in randomly I was looking for this and was testing it out. This should also support floats

`isinstance(value, (int, float))`